src/matter/cluster/ScenesCluster.ts

```diff
@@ -0,0 +1,295 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+    ArrayT,
+    BooleanT,
+    Field,
+    OptionalField,
+    ObjectT,
+    StringT,
+    UInt8T,
+    UInt16T,
+    UInt32T,
+    Bound,
+    EnumT, BitMapT, Bit
+} from "../../codec/TlvObjectCodec";
+import { Attribute, OptionalAttribute, Cluster, Command, OptionalCommand, NoResponseT } from "./Cluster";
+import { StatusCode } from "../interaction/InteractionMessages";
+import { GroupIdT } from "../common/GroupId";
+import { ClusterIdT } from "../common/ClusterId";
+import { NodeIdT } from "../common/NodeId";
+import { AttributeIdT } from "../common/AttributeId";
+import { MatterApplicationClusterSpecificationV1_0 } from "../../Specifications";
+
+/* TODO
+<bitmap name="ScenesCopyMode" type="BITMAP8">
```

Comment:
Bitmaps are implemented now:
const ScenesCopyMode = BitmapT({
   copyAllScenes: Bit(0),
});

Reply:
missed this one. added now